DEPS

```diff
@@ -110,7 +110,7 @@ allowed_hosts = [
 ]
 
 deps = {
-  'src': 'https://github.com/flutter/buildroot.git' + '@' + 'edf6aa45cbd56aa0bdd9d9333f2ac8e2d29a8be8',
+  'src': 'https://github.com/flutter/buildroot.git' + '@' + '73cd40aa9d6b76aff637267e02fda6f6ce571c41',
```

Comment:
Wanna roll this one further to include the last fixes as well?

Reply:
Done